### PR TITLE
Fix UserModelModules behavior

### PR DESCRIPTION
**Pull request by acs-ferreira**

### CODE REVIEW

File: protected/humhub/modules/user/behaviors/UserModelModules.php [hunk 1]
```diff
@@ -24,8 +25,8 @@
 class UserModelModules extends Behavior
 {
 
-    public $_enabledModules = null;
-    public $_availableModules = null;
+    public $enabledModules = null;
+    public $availableModules = null;
```
Comment (githubjeka) [inline protected/humhub/modules/user/behaviors/UserModelModules.php]: BC break

**Comment by acs-ferreira** [comment]
@githubjeka why this?

**Comment by githubjeka** [comment]
If some modules use `$userModelModules->_availableModules;` will be error

**Comment by acs-ferreira** [comment]
I see, but ANY of HumHub modules use them at this time.
Just for the record.

And `_` should not be used to indicate visibility.

**Comment by githubjeka** [comment]
> ANY of HumHub modules use them at this time.
This does not cancel BC

This code should be merged for 2.0.0 version HumHub, not for master (1.3dev now)

**Comment by luke-** [comment]
(Closed since this behavior is not longer available in v1.3)

**Pull request closed without merging**